components/brave_shields/core/browser/ad_block_component_installer.cc

```diff
@@ -155,6 +156,10 @@ AdBlockComponentInstallerPolicy::GetInstallerAttributes() const {
   return update_client::InstallerAttributes();
 }
 
+bool AdBlockComponentInstallerPolicy::IsBraveComponent() const {
```

Comment:
@antonok-edm 
I wonder why `AdBlockComponentInstallerPolicy` doesn't implement `BraveComponentInstallerPolicy`?

Reply:
to me the `BraveComponent` and `BraveComponentInstallerPolicy` look like a legacy thing that should be cleaned up.

I'd like to keep this discussion out of the scope of this PR.

Reply:
> to me the `BraveComponent` and `BraveComponentInstallerPolicy` look like a legacy thing that should be cleaned up.

Yep, this code is from before my time at Brave and I have no attachment to it. But agreed that it doesn't need to be handled here.